src/content/en/updates/2019/07/image-support-for-async-clipboard.md

```diff
@@ -14,14 +14,14 @@ description: Starting in Chrome 76, the async clipboard API now handles (some) i
 {% include "web/_shared/contributors/thomassteiner.html" %}
 
 In Chrome 66, we shipped the [text portion](/web/updates/2018/03/clipboardapi)
-of the Asynchronous [Clipboard API](https://w3c.github.io/clipboard-apis/).
-Now in Chrome 76, adding support for images to the Asynchronous
-Clipboard API, making it easy to programmatically copy and paste
-`image/png` images.
+of the [Asynchronous Clipboard
+API](https://w3c.github.io/clipboard-apis/#async-clipboard-api). Now in Chrome
+76, adding support for images makes it easy to programmatically copy and paste
+png images.
 
 <aside class="caution">
   <b>Note:</b>
-  At the time of writing, only <code>image/png</code> files are supported.
+  At the time of writing, only png files are supported.
```

Comment:
When referring generically to a file type don't use the MIME type. 

Is this note still true?

Reply:
This is still true, yes.

Reply:
Oh, and thanks for the style note!